src/arm/arm.c

```diff
@@ -73,7 +73,7 @@ mraa_arm_platform()
                     else if (mraa_file_exist("/sys/class/leds/green:ph24:led1")) {
                         platform_type = MRAA_BANANA;
                     }
-                } else if (strstr(line, "DE0/DE10-Nano-SoC")) {
+                } else if (strstr(line, "DE0/DE10-Nano-SoC") || strstr(line, "Altera SOCFPGA")) {
```

Comment:
Please add this as a separate `else if` statement to align with the code style used here.

Reply:
I tried to fix problems that you mentioned. If something is wrong, let me know. Thanks for helping and being courteous. I accidentally added 1 wrong commit but I fixed.